README.md

```diff
@@ -8,18 +8,24 @@
 * Supports dbt version `1.3.*`
```

Comment:
Could you update the https://github.com/dbt-athena/dbt-athena#iceberg section of the readme?
There is an example that need to be tweaked, specifically we use `partitioned_by=['bucket(5, user_id)'],` that no longer works, but instead should be `partitioned_by=['bucket(user_id,5)'],` so Athena is not consistent accross ctas and table creation. This will avoid some headache when people want to test an iceberg model from the example.